media/css/firefox/lockwise/lockwise.scss

```diff
@@ -6,52 +6,62 @@ $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
 @import '../../../protocol/css/includes/lib';
-@import '../../../protocol/css/components/hero';
 @import '../../../protocol/css/components/picto';
+@import '../../../protocol/css/components/split';
 @import '../../../protocol/css/templates/card-layout';
 @import '../../../protocol/css/templates/multi-column';
+@import '../../../protocol/css/components/logos/wordmark';
+@import '../../../protocol/css/components/logos/wordmark-product-lockwise';
 
+// split
 
-.add-on-download p {
-    margin: $spacing-sm;
+.mzp-c-split {
+    padding: $spacing-2xl;
 }
 
-.card-header {
-    text-align: center;
+.mzp-c-split-container {
+
+    p {
+        font-size: 18px;
+    }
+
+    @media #{$mq-lg} {
+        box-sizing: content-box;
+    }
 }
 
-.ios-button {
-    text-decoration: none;
+.mzp-c-split-body .mzp-u-title-md {
+    @include font-base;
+    font-size: 38px;
 }
 
-.links {
-    margin: $layout-xl auto;
-    text-align: center;
+@media #{$mq-lg} {
 
-    li {
-        display: inline-block;
-        margin: 0 $spacing-md;
+    .mzp-c-split-body .mzp-u-title-md {
+         font-size: 38px;
```

Comment:
Try and avoid arbitrary font sizes if possible. I'd suggest taking a look at the typography scale in Protocol and trying to use the set mixins: https://protocol.mozilla.org/fundamentals/typography.html#type-scale